### PR TITLE
DO you have discord

### PR DESCRIPTION
**Pull request by ElectroGamesDev**
DO you have discord

### CODE REVIEW

**Comment by ZandercraftGames** [comment]
Why the naming though?

File: .poggit.yml [hunk 1]
```diff
@@ -1,7 +1,7 @@
---- # Poggit-CI Manifest. Open the CI at https://poggit.pmmp.io/ci/GamakCZ/SkyWars
+--- # Poggit-CI Manifest. Open the CI at https://poggit.pmmp.io/ci/hachkingtohach1/SkyWarsTeam_Draz
 branches:
 - master
 projects:
-  SkyWars:
+  SkyWarsTeams:
     path: SkyWars/
```
Comment (GamakCZ) [inline .poggit.yml]: If you want to merge this pull rq, Move all SW team files to new dir

File: README.md [hunk 2]
```diff
@@ -1,198 +1 @@
...
-
-
-<div align="center">
-    <h2>Credits</h2>
-</div>
-
-<div align="center">
-    - Icon made by <a href="http://www.freepik.com" title="Freepik">Freepik</a> from <a href="https://www.flaticon.com/" title="Flaticon">www.flaticon.com</a> is licensed by <a href="http://creativecommons.org/licenses/by/3.0/" title="Creative Commons BY 3.0" target="_blank">CC 3.0 BY</a>
-</div>
+# SkyWars Team for PocketMine-PMMP
```
Comment (GamakCZ) [inline README.md]: Merge the description of SkyWars and SWT

File: SkyWars/plugin.yml [hunk 3]
```diff
@@ -1,28 +1,30 @@
-name: SkyWars
+name: SkyWarsTeam
 api: 3.0.0
-version: 1.2.1
+version: 1.0.1_BUILDING
```
Comment (GamakCZ) [inline SkyWars/plugin.yml]: It's better to use 1.0.1-beta1

File: SkyWars/plugin.yml [hunk 4]
```diff
@@ -1,28 +1,30 @@
...
 main: vixikhd\skywars\SkyWars
 author: VixikHD
-description: SkyWars minigame plugin
+dev: DragoVN
```
Comment (GamakCZ) [inline SkyWars/plugin.yml]: you can use
`authors: [VixikHD, DragoVN]`

File: SkyWars/plugin.yml [hunk 5]
```diff
@@ -1,28 +1,30 @@
...
+description: SkyWars Team minigame plugin
 website: https://github.com/GamakCZ/SkyWars
+website_skywars_team: https://github.com/hachkingtohach1/SkyWarsTeam
```
Comment (GamakCZ) [inline SkyWars/plugin.yml]: After merge this could be in same repo

File: SkyWars/src/vixikhd/skywars/SkyWars.php [hunk 6]
```diff
@@ -85,11 +85,22 @@ public function onChat(PlayerChatEvent $event) {
                 "§7help : Displays list of available setup commands\n" .
                 "§7slots : Updates arena slots\n".
                 "§7level : Sets arena level\n".
+                "§7maxslotsteam : Sets max slots for teams arena level\n".
                 "§7spawn : Sets arena spawns\n".
                 "§7joinsign : Sets arena join sign\n".
                 "§7savelevel : Saves the arena level\n".
                 "§7enable : Enables the arena");
                 break;
+            case "maxslotsteam":
```
Comment (GamakCZ) [inline SkyWars/src/vixikhd/skywars/SkyWars.php]: I'd rather name it 'playersperteam'

File: SkyWars/src/vixikhd/skywars/SkyWars.php [hunk 7]
```diff
@@ -85,11 +85,22 @@ public function onChat(PlayerChatEvent $event) {
...
+                if(!isset($args[1])) {
+                    $player->sendMessage("§cUsage: §7maxslotsteam <int>");
+                    break;
+                }
+                $arena->data["slots_reds"] = (int)$args[1];
```
Comment (GamakCZ) [inline SkyWars/src/vixikhd/skywars/SkyWars.php]: Make just one index for players per team, it's useless to store 3x same value

File: SkyWars/src/vixikhd/skywars/SkyWars.php [hunk 8]
```diff
@@ -85,11 +85,22 @@ public function onChat(PlayerChatEvent $event) {
...
+                $arena->data["slots_greens"] = (int)$args[1];
+                $arena->data["slots_blues"] = (int)$args[1];
+                $player->sendMessage("§a> Max slots for teams updated to $args[1]!");
```
Comment (GamakCZ) [inline SkyWars/src/vixikhd/skywars/SkyWars.php]: Max players per team

File: SkyWars/src/vixikhd/skywars/arena/Arena.php [hunk 9]
```diff
@@ -106,6 +111,21 @@ public function __construct(SkyWars $plugin, array $arenaFileData) {
         }
     }
 
+    public function getTeamsRandom($player){
+		$randomteam = mt_rand(0,2);
```
Comment (GamakCZ) [inline SkyWars/src/vixikhd/skywars/arena/Arena.php]: just rand()

File: SkyWars/src/vixikhd/skywars/arena/Arena.php [hunk 10]
```diff
@@ -83,6 +83,11 @@ class Arena implements Listener {
 
     /** @var Level $level */
     public $level = null;
+    
+    // 5 Teams for plugin
+    public $reds = [];
```
Comment (GamakCZ) [inline SkyWars/src/vixikhd/skywars/arena/Arena.php]: Look to my EggWars plugin how to make teams easier.... Having array for each team doesn't look good

File: SkyWars/src/vixikhd/skywars/arena/Arena.php [hunk 11]
```diff
@@ -119,6 +139,31 @@ public function joinToArena(Player $player) {
             $player->sendMessage("§c> Arena is full!");
             return;
         }
+	    
+	$this->getTeamsRandom($player);
+        
+        if(count($this->reds) >= $this->data["slots_reds"]) {
```
Comment (GamakCZ) [inline SkyWars/src/vixikhd/skywars/arena/Arena.php]: Too complicate code

File: SkyWars/src/vixikhd/skywars/arena/Arena.php [hunk 12]
```diff
@@ -197,9 +260,25 @@ public function startGame() {
         foreach ($this->players as $player) {
             $players[$player->getName()] = $player;
             $player->setGamemode($player::SURVIVAL);
-        }
-
-
+            
+            foreach ($this->reds as $reds){
```
Comment (GamakCZ) [inline SkyWars/src/vixikhd/skywars/arena/Arena.php]: A lot of confused code

File: SkyWars/src/vixikhd/skywars/arena/Arena.php [hunk 13]
```diff
@@ -519,6 +619,9 @@ private function createBasicData() {
             "level" => null,
             "slots" => 12,
             "spawns" => [],
+            "slots_reds" => 0,
```
Comment (GamakCZ) [inline SkyWars/src/vixikhd/skywars/arena/Arena.php]: "playersperteam" instead "slots_reds" etc.

**Review by GamakCZ — COMMENTED** [review]
Needs a lot of changes

**Pull request closed without merging**